pkg/configuration/configuration.go

```diff
@@ -222,11 +223,27 @@ func (ev *extendedViper) Set(key string, value interface{}) {
 	}
 }
 
+func bindEnvSnykErr(key string, err error) snyk_errors.Error {
+	return snyk_errors.Error{
+		Title:          "Unable to bind env",
+		Classification: "ACTIONABLE",
```

Comment:
Question: how is it possible for a user to action on this error?

Reply:
I think given the feedback below, it's not necessary to have an error catalog error here?